Examples/Detectors/DD4hepDetector/src/GeoIDHooks.cpp

```diff
@@ -0,0 +1,30 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2017 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "ActsExamples/DD4hepDetector/GeoIDHooks.h"
+
+namespace det {
+namespace geoIDHook {
+
+Acts::GeometryIdentifier stripEndcapODD(Acts::GeometryIdentifier identifier,
+                                        const Acts::Surface& surface) {
+  if (identifier.volume() == 28 || identifier.volume() == 30) {
```

Comment:
would it be possible to decide on this based on dd4hep information?

same with the radius below

Reply:
Hmm you mean e.g. by name? I'm not actually sure, and even if this is somehow structured in the DD4hep description I don't think we need to be overly generic here. This is a geometry specific problem, and I think a geometry specific solution is ok for that...

Reply:
I think theoretically, one would be able to unpack the dd4hep specific detector element from the surface instance.

Reply:
sure I would also not go too far with this but if the IDs ever change this might be a funny thing to debug

Reply:
Hmm in any case I would write some kind of python test that checks that and that should also fail if the geometry IDs change...
But if its easy to do in a more robust way, then even better.

Reply:
To be fair if the ID changes all the Digitisation+MaterialMap+... would break so this won't be the biggest issue. On the other hand if there is a more elegant solution I am all for it !